pages/linux/runuser.md

```diff
@@ -0,0 +1,23 @@
+# runuser
+
+> Run commands as a specific user and group IDs. Does not ask for password. Only useful when run as the root user.
```

Comment:
```suggestion
> Run commands as a specific user and group without asking for password (needs root privileges).
```

Now that I take a closer look at it, I think this can be simplified in a shorter sentence. What do you think?

Reply:
Agree, thanks for reviewing.